src/components/home/ChatRoomCard/index.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * 채팅방 카드 컴포넌트
+ * @author 선우
+ */
+
+import Tag from '@/components/@common/Tag';
+import * as Styled from './ChatRoomCard.styled';
+import { tags } from '@/constants/tag';
+
+export interface IChatRoomCardProps {
+  id: number;
+  title: string;
+  imageUrl: string;
+  categoryIdList: {
+    id: number;
+  }[];
+  participantCount: number;
+  participantProfileList: string[];
+}
+
+const ChatRoomCard = (chatRoom: IChatRoomCardProps) => {
+  return (
+    <Styled.Wrapper>
+      <Styled.ImageContainer>
+        <Styled.ChatRoomThumbnail src={chatRoom.imageUrl} />
+      </Styled.ImageContainer>
+      <Styled.ChatRoomInfoContainer>
+        <Styled.ChatRoomTitle>{chatRoom.title}</Styled.ChatRoomTitle>
+        <Styled.ChatRoomSubtitle>채팅방 부제목 영역</Styled.ChatRoomSubtitle>
+        <Styled.ChatRoomTagSection>
+          {chatRoom.categoryIdList.map((category, idx) => {
```

Comment:
```suggestion
          {chatRoom.categoryIdList.slice(0, 3).map((category, idx) => {
```

이런식으로 배열을 잘라서 사용하는 방법은 어떤가요?? 저는 길이가 짧아서 성능부분은 신경안써도 괜찮을 것 같은데 코드 가독성에서 더 괜찮다고 느껴지는데 의견 주시면 감사하겠습니다!!

Reply:
오호 이렇게 하면 코드가 훨씬 간결해지네요! 좋아욥 👍